lib/tasks/TaskUtil.js

```diff
@@ -108,6 +108,18 @@ class TaskUtil {
 		return this._projectBuildContext.isRootProject();
 	}
 
+	/**
+	 * Retrieves a build option defined by it's <code>key</code.
+	 * If no option with the given <code>key</code> is stored, <code>undefined</code> is returned.
+	 *
+	 * @param {string} key The option key
+	 * @returns {object|undefined} The build option (or undefined)
```

Comment:
The option value can be of any type, right?
```suggestion
	 * @returns {any|undefined} The build option (or undefined)
```

Reply:
thats true. While introducing `BuildOptionsCollection` we could limit the kind of values